src/modules/commentPage.js

```diff
@@ -0,0 +1,44 @@
+import fetchComment from './fetchComment.js';
+
+const body = document.querySelector('body');
+const detailContainer = document.createElement('div');
+
+export const commentPage = async (objectDetails, loops) => {
+  const comments = await fetchComment(objectDetails.objectID);
+
+  detailContainer.innerHTML = `
+    <section id="card">
+        <p id='Close'>X</p>
+        <image class="imgContainer" src = "${objectDetails.primaryImage}" alt="Object Image"/>
+        <h2 class='title1 detail'>${objectDetails.title}</h2>
+        <p class='detail1'>Artist Display Name: ${objectDetails.artistDisplayName}</p>
+        <p class='detail2'>Dimensions: ${objectDetails.dimensions}</p>
+        <p class='detail1'>Object name: ${objectDetails.objectName}</p>
+        <p class='detail2'>Object Date: ${objectDetails.objectDate}</p>
+        <p class='detail1'>City: ${objectDetails.city}</p>
+        <p class='detail2'>Country: ${objectDetails.country}</p>
+        <p id='dComment'>Comments (${comments.length})</p>`;
```

Comment:
For the test case to work, it is better if you create a separate function for the comment counter.